app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_dict in people:
+        person = Person(person_dict["name"], person_dict["age"])
+        person_list.append(person)
+
+    for person in person_list:
+        for person_dict in people:
+            if person_dict["name"] == person.name:
+                if person_dict.get("wife") is not None:
+                    person.wife = Person.people[person_dict["wife"]]
+                if person_dict.get("husband") is not None:
+                    person.husband = Person.people[person_dict["husband"]]
```

Comment:
Consider simplifying the nested `if` statements by using logical operators. For example, you could combine the checks for 'wife' and 'husband' using `if person_dict.get('wife') or person_dict.get('husband'):` and then handle each case within the block. This would improve the readability of your code.